spec/rubocop/cop/layout/empty_line_after_guard_clause_spec.rb

```diff
@@ -106,6 +106,36 @@ def foo
     RUBY
   end
 
+  it 'registers an offense for guard clause followed by empty line' \
+     'when guard clause including heredoc' do
+    expect_no_offenses(<<-RUBY.strip_indent)
+      def method
+        if truthy
+          raise <<-MSG
+            This is an error.
+          MSG
+        end
+
+        value
+      end
+    RUBY
+  end
+
+  it 'registers an offense for guard clause not followed by empty line' \
+     'when guard clause including heredoc' do
+    expect_offense(<<-RUBY.strip_indent)
+      def method
+        if truthy
+          raise <<-MSG
+            This is an error.
+          MSG
+        end
+        ^^^ Add empty line after guard clause.
```

Comment:
We're stretching the definition of *guard clause* a bit, but I would also want a space here, so I don't mind. 😬

Reply:
I can live with this. Let's see how the users will respond to the change.